src/components/SideNavMenu.tsx

```diff
@@ -0,0 +1,83 @@
+'use client';
+
+import { useState } from 'react';
+import { usePathname } from 'next/navigation';
+import Image from 'next/image';
+import Link from 'next/link';
+import ProfileImage from '@/components/ProfileImage'; // 기존 프로필 이미지 컴포넌트 사용
+
+// SVG 아이콘
+import MyInfo from '@/assets/icons/my-users.svg'; // 내정보
+import MyReservation from '@/assets/icons/my-reservations.svg'; // 예약내역
+import MyActivities from '@/assets/icons/my-activities.svg'; // 내 체험 관리
+import MyReservationStatus from '@/assets/icons/my-activities-dashboard.svg'; // 예약 현황
+import EditIcon from '@/assets/icons/pencil.svg';
+
+const NAV_ITEMS = [
+  { name: '내 정보', path: '/my-page', icon: MyInfo },
+  { name: '예약 내역', path: '/my-reservations', icon: MyReservation },
+  { name: '내 체험 관리', path: '/my-activities', icon: MyActivities },
+  { name: '예약 현황', path: '/reservation-status', icon: MyReservationStatus },
+];
+
+const SideNavMenu = () => {
+  const pathname = usePathname();
+  const [previewImage, setPreviewImage] = useState<string | null>(null);
+
+  // 파일 선택 핸들러
+  const handleFileChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+    const file = event.target.files?.[0];
+    if (file) {
+      const imageUrl = URL.createObjectURL(file);
+      setPreviewImage(imageUrl);
+    }
+  };
+
+  return (
+    <aside className='w-[251px] max-w-[251px] flex-none rounded-lg border border-gray-300 bg-white p-4 md:w-[384px] md:max-w-[384px]'>
```

Comment:
여백 체크해 주시고, width 값을 설정하면 max-width 값은 무시돼 버리기 때문에 빼셔도 될 것 같습니다! 또 그림자 넣어주셔야 합니다~
```suggestion
    <aside className='w-[251px] flex-none rounded-lg border border-gray-300 bg-white p-6 md:w-[384px]'>
    ```